corehq/apps/data_dictionary/util.py

```diff
@@ -1,13 +1,17 @@
+from corehq import toggles
 from corehq.apps.app_manager.dbaccessors import get_case_types_from_apps
 from corehq.apps.app_manager.util import all_case_properties_by_domain
 from corehq.apps.data_dictionary.models import CaseProperty, CaseType
 from corehq.apps.export.models.new import CaseExportDataSchema
 
 
 def generate_data_dictionary(domain):
+    if toggles.OLD_EXPORTS.enabled(domain):
```

Comment:
@benrudolph I added this check, but from our last conversation it may not actually be necessary right?